### DISCUSSION TITLE
チーム名！

### DISCUSSION THREAD

**Discussion by Terralien-jp** [Hackathon5-TeamD/evolution-quest#26] [General]
# チーム名募集！
* Extend D（小林さん案）
など

**Comment by purewest-github** [comment]
寺組

**Reply by Terralien-jp** [reply]
西組でｗ

**Comment by kobakichi** [comment]
KSI_Developers

Kindness   優しさ
Strict  厳格さ
Interesting   面白さ

という一面を持っているメンバーが揃っているのではというイメージです！

**Reply by Terralien-jp** [reply]
もういっこSつけてKISS_DEVなんてのもいいかもですね！（完全にネタなのでスルーしてくださいｗ）

**Comment by Terralien-jp** [comment]
思うにハッカソン単位のチーム名もいいのですが、OSS活動として継続的なものもあってもいいのかもですね(?_?)
OSS同好会の立場からするとハッカソンはあくまで日頃の行動の発表会でしかないみたいなｗ
もっと少人数でチーム単位で切磋琢磨したいって場合はハッカソン時はどこかチームに所属するなど？
